docs/index.rst

```diff
@@ -28,6 +28,15 @@ Welcome to Concept to Clinic's documentation!
    code-of-conduct-link
    contributor-license-agreement-link
 
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Code Documentation
```

Comment:
This section shouldn't be named Code Documentation probably now that autodoc has been removed?